curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/es6/explore-differences-between-the-var-and-let-keywords.portuguese.md

```diff
@@ -2,27 +2,59 @@
 id: 587d7b87367417b2b2512b3f
 title: Explore Differences Between the var and let Keywords
 challengeType: 1
-videoUrl: ''
-localeTitle: Explore as diferenças entre o var e let
+forumTopicId: 301202
```

Comment:
There will be a completely separate Portuguese forum at some point that will have its own translated topics.  We do not want to use the English version here, so this change should be reverted.